ts/editor/SecondaryInputBadge.svelte

```diff
@@ -37,20 +38,30 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 >
     <Badge
         tooltip="{tr.editingToggleHtmlEditor()} ({getPlatformString(keyCombination)})"
-        iconSize={80}
-        --icon-align="text-top">{@html icon}</Badge
+        iconSize={75}>{@html icon}</Badge
     >
 </span>
 
 <style lang="scss">
     span {
-        opacity: 0.4;
-
+        cursor: pointer;
+        opacity: 0;
         &.highlighted {
             opacity: 1;
         }
+    }
+    :global(.editor-field) {
+        &:focus-within,
```

Comment:
I'd prefer to define such variables on the *higher level* (e.g. _props.scss from https://github.com/ankitects/anki/pull/1989). The 0.4 - 0.8 - 1.0 steps are used for all badges, so I'd like to define them in one place.

This will be done in one sweep with a coming up PR.

I went for a `visible` boolean that is passed down from `NoteEditor.svelte`: https://github.com/ankitects/anki/pull/2002/commits/37472dcea757cc1e3b55413d4933a1b82670b62e.